app/assets/stylesheets/application.css

```diff
@@ -223,8 +223,10 @@ div.shorten_first_p p:first-child {
 }
 
 hr {
+	/* make it lighter than the browser's default */
 	border: none;
-	border-top: 1px dotted var(--color-fg-shape);
+	height: 1px;
+	background-color: #BBB;
```

Comment:
Please use one of the existing custom properties for colors or create one. That'll make sure it supports both light and dark mode, and doesn't accidentally get edited in a way that loses contrast.